src/Microsoft.ManagedZlib/tests/LocalMemoryStream.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This whole file could be removed, it's not being used anywhere (except in the commented test, which we probably want to remove too).